lab_share_lib/processing/rabbit_message_processor.py

```diff
@@ -33,10 +45,21 @@ def _build_processor_for_subject(self, subject: str) -> BaseProcessor:
             BaseProcessor, processor_instance_builder(self._schema_registry, self._basic_publisher, self._config)
         )
 
+    def build_avro_encoder(self, encoder_type, subject):
+        if encoder_type not in ENCODERS.keys():
+            raise Exception(f"Encoder type {encoder_type} not defined")
+
+        return ENCODERS[encoder_type](self._schema_registry, subject)
+
+    def encoder_type(self, headers):
+        if RABBITMQ_HEADER_KEY_ENCODER_TYPE not in headers.keys():
+            return RABBITMQ_HEADER_VALUE_ENCODER_TYPE_DEFAULT
+        return headers[RABBITMQ_HEADER_KEY_ENCODER_TYPE]
```

Comment:
This seems like duplication of the method of the same name in `lab_share_lib/processing/rabbit_message.py` - is it possible to use that instead? (does the rabbit message exist at this point?)

Reply:
Good spot, I'll change it